js/samples/04.ai.b.messageExtensions.AI-ME/README.md

```diff
@@ -111,6 +109,12 @@ app.messageExtensions.submitAction<SubmitData>('CreatePost', async (context: Tur
     cd teams-ai/js/samples/04.ai.b.messageExtensions.aime/
     ```
 
+4. Duplicate the `sample.env` in the `teams-ai/js/samples/04.ai.b.messageExtensions.AI-ME` folder. Rename the file to `.env`. 
+
+5. If you are using OpenAI then only keep the `OPENAI_KEY` and add in your key. Otherwise if you are using AzureOpenAI then only keep the `AZURE_OPENAI_KEY`, `AZURE_OPENAI_ENDPOINT`variables and fill them in appropriately.
+
+6. Update `config.json` and `index.ts` with your model deployment name.
```

Comment:
This only applies to AOAI right? It might be helpful to add a screencap of where that name is. 

Name, not number?

Reply:
It applies to both, and name is the term